app/src/test/java/io/gnosis/safe/ui/transaction/TransactionsViewModelTest.kt

```diff
@@ -0,0 +1,147 @@
+package io.gnosis.safe.ui.transaction
+
+import io.gnosis.data.models.*
+import io.gnosis.data.models.TransactionStatus.*
+import io.gnosis.data.repositories.SafeRepository
+import io.gnosis.data.repositories.TokenRepository.Companion.ETH_SERVICE_TOKEN_INFO
+import io.gnosis.data.repositories.TransactionRepository
+import io.gnosis.safe.R
+import io.gnosis.safe.TestLifecycleRule
+import io.gnosis.safe.TestLiveDataObserver
+import io.gnosis.safe.appDispatchers
+import io.gnosis.safe.ui.base.BaseStateViewModel
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.mockk
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertTrue
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import pm.gnosis.model.Solidity
+import pm.gnosis.utils.asEthereumAddress
+import java.math.BigInteger
+
+class TransactionsViewModelTest {
+    @get:Rule
+    val instantExecutorRule = TestLifecycleRule()
+
+    private val safeRepository = mockk<SafeRepository>()
+    private val transactionRepository = mockk<TransactionRepository>()
+    private lateinit var viewModel: TransactionsViewModel
+    private val defaultSafeName: String = "Default Name"
+    private val defaultSafeAddress: Solidity.Address = "0x1234".asEthereumAddress()!!
+    private val defaultAddress: Solidity.Address = Solidity.Address(BigInteger.ZERO)
+    private val defaultSafe = Safe(defaultSafeAddress, defaultSafeName)
```

Comment:
we have put test constants to companion object in other test classes. This is not a blocker, but we should follow same structure for tests

Reply:
Sorry, I merged prematurely. I will keep that in mind.